app/controllers/api/experimental/concerns/column_data.rb

```diff
@@ -180,6 +180,7 @@ def sum_columns(column_names, work_packages)
     end
   end
 
+  # FIXME: Deprecated. Should be removable once the experimental WP controller is gone
```

Comment:
So was the comment simply wrong are are you going to remove the method later in this PR? :wink: